### PR TITLE
Unit testing2

### PR DESCRIPTION
**Pull request by AmineHLub**
<h1> Summary </h1>

- Added mocks and tests for checkbox status updating, and input updating functions.
- The DeleteAll function is already since the first project (unit testing part 1)
- Added Describe function in testing

### CODE REVIEW

File: src/__tests__/mockdelete.test.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 const removeAll = require('../__mocks__/mockdelete.js');
 
-test('Removing according to dummyArr 2 false status did not work', () => {
-  expect(removeAll()).toBe(1);
+describe('Test (All)removing functions', () => {
+  test('Removing according to dummyArr 2 false status did not work', () => {
+    expect(removeAll()).toBe(1);
+  });
 });
```
Comment (george-shammar) [inline src/__tests__/mockdelete.test.js:7]: You've done a really nice job with your tests and your code is easily understandable. Good job. :clap: 

Let's take a look at the following required changes in order to make it even better.

- [ ] Please clean up the commented codes you have inside `mockdeleteSingle.test.js`.
- [ ] Kindly include test for the function for editing the task description.

[OPTIONAL]

- [ ] Please consider adding details of your test to the README and how to execute the test.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @AmineHLub , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- See comments below.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/mockdelete.test.js (hunk 1 above)
Comment (ahmed-alfarouq) [reply src/__tests__/mockdelete.test.js:7]: Hi sir, 
we already did the second change you want from us the file name is updater.test.js.
Hope you will clear exactly what do you want from us, please?
Thanks
@george-shammar

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi Team,

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**